server/public/model/slack_attachment.go

```diff
@@ -218,15 +219,18 @@ func (s *SlackAttachmentField) Equals(input *SlackAttachmentField) bool {
 		return false
 	}
 
-	if s.Value != input.Value {
-		return false
-	}
-
-	if s.Short != input.Short {
-		return false
+	switch s.Value.(type) {
+	case string, bool, int, float64:
+		if s.Value != input.Value {
+			return false
+		}
+	default:
+		if !reflect.DeepEqual(s.Value, input.Value) {
+			return false
+		}
```

Comment:
Would it maybe be better to test whether or not the value is comparable, and if not use DeepEqual?